makefile

```diff
@@ -41,6 +41,23 @@ CXX = $(CC)
 include make/tests    # tests
 include make/cpplint  # cpplint
 
+
+## FIXME(dl): move to appropriate place later
+## GPU settings
+
+ifdef STAN_GPU
+  CXXFLAGS += -isystem $(OPENCL) -DSTAN_GPU -DSTAN_DEVICE_CPU
+  ifeq ($(shell echo $(TRAVIS_OS_NAME)),linux)
+    LDLIBS+=-L/home/travis/AMDAPPSDK/lib/x86_64/
+  endif
+
+  ifeq ($(OS_TYPE),mac)
+    LDLIBS += -framework OpenCL
+  else
+    LDLIBS += -lOpenCL
+  endif
+endif
```

Comment:
Odd, this seems to not catch in travis, but I'll look at it tonight

https://travis-ci.org/stan-dev/math/jobs/331837522#L1025